Frontend/src/Components/Graph/CustomGraph.js

```diff
@@ -63,18 +59,23 @@ for (const category of GraphData.Output) {
   }
 }
 
+
 // the options fed into the graph object, save regardless of datasets
```

Comment:
```suggestion

// the options fed into the graph object, save regardless of datasets
```
Fixing unnecessary whitespace change

Reply:
Addressed